Cargo.toml

```diff
@@ -10,21 +10,24 @@ categories = ["rendering", "external-ffi-bindings", "game-engines", "multimedia"
 keywords = ["assimp", "3d", "blend", "3ds", "glTF"]
 repository = "https://github.com/jkvargas/russimp-sys"
 description = "Raw Assimp bindings for rust"
-exclude = ["*.bash", "*.ps1"]
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+exclude = ["/assimp"]
 
 [lib]
 name = "russimp_sys"
 path = "src/lib.rs"
 
-[build-dependencies]
-bindgen = "0.59.1"
-vcpkg = "0.2.15"
```

Comment:
I'd rather keep vcpkg as an option as well.
vcpkg has a pull request to update assimp to the latest version

Reply:
Yes, I just put it back.